### ISSUE TITLE
[BUG] Not parsing MyClipinngs.txt

### ISSUE THREAD

**Issue by edunet22** [hadynz/obsidian-kindle-plugin#140]
**Describe the bug**
My obsidian plugin is not parsing the MyClippings.txt file.

**To Reproduce**
When I try to do import the file it is not working, gives an TypeError failure.
My kindle's file starts with (breakline) and =======. 
Example: 
```

==========
Jornada de Montahue - P Hernan Alessandri 
- Your Highlight on Page 125-125 | Added on Tuesday, April 24, 2012, 04:13 PM

para nosotros el fondo de la realidad,lo másreal de todo,no son las fuerzas de producción,no es laeconomía,no es la materia,sino Dios.Y Dios es un ser
==========
Jornada de Montahue - P Hernan Alessandri 
- Your Highlight on Page 125-125 | Added on Tuesday, April 24, 2012, 04:13 PM

personal
==========
```
When I delete every character until the first letter it works.

**Desktop (please complete the following information):**
 - OS:Linux
 - Plugin version  1.5.2
 - Obsidian version 0.13.19

**Comment by marc0l92** [comment]
The file is not in the expected format. The plugin expects to find the file in the format:
- title
- metadata
- space
- note
- `==========`

but your file starts with "space" and the line.

If you remove the first two lines the plugin works:
```
Jornada de Montahue - P Hernan Alessandri 
- Your Highlight on Page 125-125 | Added on Tuesday, April 24, 2012, 04:13 PM

para nosotros el fondo de la realidad,lo másreal de todo,no son las fuerzas de producción,no es laeconomía,no es la materia,sino Dios.Y Dios es un ser
==========
Jornada de Montahue - P Hernan Alessandri 
- Your Highlight on Page 125-125 | Added on Tuesday, April 24, 2012, 04:13 PM

personal
==========
```

Anyway I would keep this issue open in order to make the script more robust and handle automatically those exceptions.

**Comment by hadynz** [comment]
Thanks for raising this rather important bug!

Fixed in [1.5.3][1]. If you can test and confirm that it is working for you, that would be great. 

[1]: https://github.com/hadynz/obsidian-kindle-plugin/releases/tag/1.5.3
